commons/src/main/com/mbrlabs/mundus/commons/utils/MathUtils.java

```diff
@@ -26,6 +27,40 @@
 public class MathUtils {
     private static final Vector3 tmp = new Vector3();
 
+    public final static Pool<Vector2> vector2Pool = new Pool<Vector2>() {
```

Comment:
What do you think to move this pools to separate util file, like PoolsUtils?

Reply:
Sure, moved them to a "Pools" class